tests/conftest.py

```diff
@@ -383,6 +400,7 @@ class ComputerFirecrestConfig:
     temp_directory: str
     workdir: str
     api_version: str
+    builder_metadata_options_account: str
```

Comment:
Maybe you can add a small docstring for this class, this variable is not so clear to me. I think most variables are explained in the README and can copy pasted to here.

Reply:
Alright, docstring added. 

`builder_metadata_options_custom_scheduler_commands` are "custom" settings that users had to define while submitting jobs. These are cluster specific variables and historically people using aiida were to set these variables in their workflow, etc..
an example of this variable is like this: 
`["#SBATCH --account=mr32", "#SBATCH --constraint=mc", "#SBATCH --mem=10K"]`

We didn't need to do this for tests against a mocking server, but for tests against a real server these values are needed. 

Note, we don't set this for the entire computer.

Reply:
This is not so related to this PR, but for my clarification it would be nice. Usually AiiDA uses prepend-text option for adding this SBATCH commands into the script but for firecrest the SBATCH commands have to be made more explicit, therefore we have `custom_scheduler_commands`?

Reply:
Honestly I haven't explored that prepend-text variable when creating a new computer in aiida. 

But you are right, some of these could be put there, the only thing is that, then it's always the same for all jobs in that computer. 
Maybe one would want to run some jobs on `gpu` and some other `mc`, or different accounts for billing, etc..

Reply:
I think in this case you would define multiple computers, but I am not sure about best practices

Reply:
but for `mc` & `gpu`, probably I agree, I don't really know